src/modules/swap/services/safeBundleFlow/safeBundleEthFlow.ts

```diff
@@ -123,10 +124,13 @@ export async function safeBundleEthFlow(
     )
     tradeFlowAnalytics.sign(swapFlowAnalyticsContext)
 
-    logTradeFlow(LOG_PREFIX, 'STEP 8: show UI of the successfully sent transaction')
+    logTradeFlow(LOG_PREFIX, 'STEP 8: add app data to upload queue')
+    callbacks.uploadAppData({ chainId: context.chainId, orderId, appData: appDataInfo })
```

Comment:
The Safe Eth Flow is not like the EOA Eth flow.

Either it needs to upload the appData: then all SC related orders also do
Or it doesn't then it's not needed here.

Reply:
NVM, i see now what you mean! fixing it!


great catch

Reply:
https://github.com/cowprotocol/cowswap/pull/2998